connector/src/main/java/com/google/cloud/spark/bigquery/v2/BigQueryDataSourceReader.java

```diff
@@ -228,6 +232,7 @@ private ReadRowsResponseToInternalRowIteratorConverter createConverter(
 
   List<InputPartition<InternalRow>> createEmptyProjectionPartitions() {
     long rowCount = bigQueryClient.calculateTableSize(tableId, globalFilter);
+    logger.info("Creating a DataFrame of empty rows with size " + rowCount);
```

Comment:
Could we phrase this as something like "Used optimized BQ count(*) path. Count: "